Resources/public/templates/fields/edit/authorinput.hbt

```diff
@@ -15,18 +14,32 @@
                     Name:
                 </label>
                 <div class="ez-editfield-input ez-editfield-input-name"><div class="ez-author-input-ui"><input type="text"
-                    class="ez-field-author-name ez-validated-input"
-                    value="{{ author.name }}"
-                    id="ez-field-{{ content.contentId }}-{{ fieldDefinition.identifier }}-{{ author.id }}-name"
-                ></div></div>
+                                                                                                               class="ez-field-author-name ez-validated-input"
+                                                                                                               value="{{ author.name }}"
+                                                                                                               id="ez-field-{{ content.contentId }}-{{ fieldDefinition.identifier }}-{{ author.id }}-name"
+                        ></div></div>
+            </div>
+        </div>
+    </div>
+</div>
+
+<div class="pure-g ez-editfield-row">
+    <div class="pure-u ez-editfield-infos">
+        <label for="ez-field-{{ content.contentId }}-{{ fieldDefinition.identifier }}-{{ author.id }}-name">
```

Comment:
it's the label for the email, so the for attribute should `ez-field-{{ content.contentId }}-{{ fieldDefinition.identifier }}-{{ author.id }}-email`